integrated_channels/integrated_channel/management/commands/transmit_learner_data.py

```diff
@@ -20,19 +21,40 @@ class Command(IntegratedChannelCommandMixin, BaseCommand):
     Transmit Enterprise learner course completion data for the given EnterpriseCustomer.
     ''')
 
+    def add_arguments(self, parser):
+        """
+        Add required --api_user argument to the parser.
+        """
+        parser.add_argument(
+            '--api_user',
+            dest='api_user',
+            required=True,
+            metavar=_('LMS_API_USERNAME'),
```

Comment:
@e-kolpakov It's displayed to users of the management command, and isn't integral to the code, so I figured it was fair game for translating.

https://docs.python.org/3/library/argparse.html#metavar